tests/test_geminga_paper.py

```diff
@@ -14,7 +15,7 @@
 import argparse
 from collections import namedtuple
 
-
+@pytest.mark.xfail
```

Comment:
Do we really need that? Did this PR break the test?

Reply:
Geminga test is broken due to a recent(ish) change in astromodels. See https://github.com/threeML/hawc_hal/pull/19#issuecomment-481512104

Reply:
After 1.5 years, I finally got around to fixing the Geminga test in another branch.